src/events/klasaReady.ts

```diff
@@ -39,7 +40,7 @@ export default class extends Event {
 	private async initPostStatsTask() {
 		const { tasks } = this.client.schedule;
 		if (!tasks.some(task => task.taskName === 'poststats')) {
-			await this.client.schedule.create('poststats', '*/15 * * * *', {});
+			await this.client.schedule.create('poststats', '*/10 * * * *', {});
```

Comment:
```suggestion
			await this.client.schedule.create('poststats', '*/15 * * * *', {});
```

Why change this to 10?